<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptPage</class>
 <widget class="QFrame" name="ScriptPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>269</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
     <item>
      <widget class="QToolBox" name="toolBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="tabSpacing">
        <number>0</number>
       </property>
       <widget class="QWidget" name="manualScriptingpage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>374</width>
          <height>129</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Manual Scripting</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="margin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="JSEdit" name="scriptText">
           <property name="toolTip">
            <string>Javascript Filtering Rule</string>
           </property>
           <property name="statusTip">
            <string>Javascript Filtering Rule</string>
           </property>
           <property name="whatsThis">
            <string>Javascript Filtering Rule</string>
           </property>
           <property name="plainText">
            <string/>
           </property>
           <property name="tabStopWidth">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="assistedScriptingPage">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>374</width>
          <height>129</height>
         </rect>
        </property>
        <attribute name="label">
         <string>Assisted Scripting</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="0">
         <property name="margin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="QPushButton" name="addConditionButton">
               <property name="text">
                <string>Add Condition</string>
               </property>
               <property name="icon">
                <iconset resource="resources/Resources.qrc">
                 <normaloff>:/icon/AddCondition</normaloff>:/icon/AddCondition</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>16</width>
                 <height>16</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Match:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="conditionTypeCombo">
               <property name="modelColumn">
                <number>0</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="conditionsVBox" stretch="">
             <property name="spacing">
              <number>2</number>
             </property>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4"/>
     </item>
     <item>
      <widget class="QLabel" name="scriptError">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="applyButton">
         <property name="text">
          <string>Apply</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>JSEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>jsedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources/Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>applyButton</sender>
   <signal>clicked()</signal>
   <receiver>ScriptPage</receiver>
   <slot>emitApplyScript()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ScriptPage</receiver>
   <slot>emitCancelScript()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>279</x>
     <y>278</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>cancelScript()</signal>
  <signal>applyScript()</signal>
  <slot>emitCancelScript()</slot>
  <slot>emitApplyScript()</slot>
 </slots>
</ui>
